language/guidelines/user-experience/performance/index.md

```diff
@@ -0,0 +1,73 @@
+
+### Short-term and Working Memory
+
+[Short-term memory](http://en.wikipedia.org/wiki/Short-term_memory) is the capacity of holding small amount of information in an active, readily available state whereas [working memory](http://en.wikipedia.org/wiki/Working_memory) is the system responsible for holding and processing of new and already stored information. The two work together to store, comprehend, process, and update all information coming in from sensory memory as well as keep current tasks and thoughts close at hand.
+
+Short-term memory is commonly cited to only be able to hold [7 ± 2](http://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two) chunks of information at a time, and that if more information is entered in to the short-term memory in that time, retrieval of information stored earlier becomes error prone. Short term memory has also been found to decay and only last around 18 seconds unless an individual is actively trying to retain a piece of information.
+
+Working memory's capacity has been hard to pin down exactly, but the two most prominent models, when taken together, suggest that the more information that must be maintained in working memory, the slower and more error prone concurrent process become and that the rate of decay of working memory is based on the temporal density and attentional demands of the processing task (this is also known as *cognitive load*).
+
+### The Size and Speed of Immersion
+
+Sensory memory, short-term memory, and working memory combine to determine how immersive an interface feels to a user. The faster a interface is able to respond to a user, the faster that user's sensory memory can pick up the response, the faster that response can be stored in short-term memory, and the faster working memory can start processing the result in context of the input. The longer that initial step takes, or the larger the amount being asked for and retrieved, the more likely that a user's memory for the task at hand will decay. This decay is what causes *web stress* and leads users abandon tasks and sites entirely.
```

Comment:
+ "The faster **an** interface is able"
+ "and the faster that working memory can"